src/ucp/wireup/wireup_cm.c

```diff
@@ -192,35 +192,33 @@ static ssize_t ucp_cm_client_priv_pack_cb(void *arg, const char *dev_name,
 free_addr:
     ucs_free(ucp_addr);
 out:
-    if (status != UCS_OK) {
+    if (status == UCS_OK) {
+        ep->flags |= UCP_EP_FLAG_LOCAL_CONNECTED;
+    } else {
         ucp_worker_set_ep_failed(worker, ep,
                                  &ucp_ep_get_cm_wireup_ep(ep)->super.super,
                                  ucp_ep_get_cm_lane(ep), status);
     }
+
     UCS_ASYNC_UNBLOCK(&worker->async);
     return (status == UCS_OK) ? (sizeof(*sa_data) + ucp_addr_size) : status;
 }
 
-static void ucp_cm_client_connect_cb(uct_ep_h ep, void *arg,
-                                     const uct_cm_remote_data_t *remote_data,
-                                     ucs_status_t status)
+static unsigned ucp_cm_client_connect_progress(void *arg)
```

Comment:
this function is called on the client or on the server?
pls improve name or add comment

Reply:
all CBs have name like "ucp_cm_<caller_side>_foo", will add a comment